clang-tools-extra/clang-tidy/misc/IncludeCleanerCheck.cpp

```diff
@@ -199,6 +199,9 @@ void IncludeCleanerCheck::check(const MatchFinder::MatchResult &Result) {
 
   tooling::HeaderIncludes HeaderIncludes(getCurrentMainFile(), Code,
                                          FileStyle->IncludeStyle);
+  // `tooling::HeaderIncludes::insert` will not modify `ExistingIncludes`. We
+  // should handle repeat include here
+  std::set<const std::string> InsertedHeader{};
```

Comment:
`std::set<const std::string>` won't compile, please take a look at the failed CI checking. `llvm::StringSet` might be better option if `std::string` could be switched to `StringRef`.